motoman_driver/MotoPlus/CtrlGroup.c

```diff
@@ -367,13 +353,22 @@ BOOL Ros_CtrlGroup_GetFBServoSpeed(CtrlGroup* ctrlGroup, long pulseSpeed[MAX_PUL
 		registerValuesLong[i * 2] = registerValues[i * 2];
 		registerValuesLong[(i * 2) + 1] = registerValues[(i * 2) + 1];
 
-		//combine both registers into single 4 byte value (0.0001 deg/sec)
+		//combine both registers into single 4 byte value (0.0001 deg/sec or 1 um/sec)
 		dblRegister = (registerValuesLong[(i * 2) + 1] << 16) | registerValuesLong[i * 2];
 
 		//convert to pulse/sec
-		dblRegister /= 10000.0; //deg/sec
-		dblRegister *= RAD_PER_DEGREE; //rad/sec
-		dblRegister *= ctrlGroup->pulseToRad.PtoR[i]; //pulse/sec
+		if (ctrlGroup->axisType.type[i] == AXIS_ROTATION)
+		{
+			dblRegister /= 10000.0; //deg/sec

```

Comment:
Suggestion: use `1e4` (and `1e5` below)?

Reply:
Ok